weblogic-azure-aks/src/main/resources/marketing-artifacts/README.md

```diff
@@ -0,0 +1,63 @@
+<h1>Offer listing</h1>
+
+<h2>Name</h2>
+
+Oracle WebLogic Server on Azure Solution Overview
+
+<h2>Oracle WebLogic Server is a scalable, enterprise-ready Java EE application server.</h2>
+
+<h2>Description</h2>
+
+<p>Azure Oracle WebLogic Server (WLS) offers enable you to embrace cloud computing by providing greater choice and flexibility for WLS migration including lift and shift of your Java applications to Azure Virtual Machines with the smallest effort and biggest impact. The offers empower you to start your business applications quickly by automatically provisioning virtual network, storage and Linux resources, installing WLS, setting up security with a network security group, easing database connectivity, configuring load-balancing with App Gateway or Oracle HTTP Server, connecting to Azure Active Directory, enabling centralized logging via ELK as well as integrating distributed caching with Oracle Coherence.</p>
+<p>There are a number of offers that support different common use cases such as a single working instance with Admin Server enabled as well as highly available clusters.</p>
+<p>The offers are linked in the <i>Learn more</i> section at the bottom of this page. You should feel free to give them a try, the offers are available free of charge.</p>
```

Comment:
For more information about the offers, see [Learn More](https://www.oracle.com/middleware/technologies/weblogic.html).
give them a try, the offers -> give them a try; (semi-colon) the offers